### DISCUSSION TITLE
Hotkey - Reset Button (Menu +  B)

### DISCUSSION THREAD

**Discussion by zebramonkeylizard** [OnionUI/Onion#508] [Ideas]
There's no hotkey by default to reset games. it would be nice to see the option, maybe (Menu + B), and listed in the Onion Manual. I believe this feature would be used at least as much as the frame rate hotkey, if not more.

**Comment by zebramonkeylizard** [comment]
1. With Onion Launcher enabled and/or the load save state enabled in Retroarch (Onion OS default settings) your game(s) may launch in a state where you do not want to continue from and a quick reset option would be convenient. 

2. When you beat some games there's no way to continue playing without resetting them, see [https://forum.digitpress.com/forum/showthread.php?89827-Games-that-Must-Be-Reset-After-You-Beat-Them](url)

3. The Sega Genesis game X-Men requires you reset the game as part of the game itself. 

4. Reset hotkeys seem to be common in other emulation platforms, e.g., Retropie [https://retropie.org.uk/docs/Controller-Configuration/](url)

**Reply by muchitto** [reply]
This feature would be awesome! I've had some games bug out and get stuck and because of the save state, I cannot continue the game at all unless I go to the File Explorer and remove the save states manually.

**Reply by zebramonkeylizard** [reply]
Please vote for your prefered option here (https://discord.com/channels/529983248114122762/931367023588569180/1031610457057919031)

**Comment by zebramonkeylizard** [comment]
Join the discussion on discord.
